vaadinfrontend/src/main/java/life/qbic/datamanager/views/layouts/Card.java

```diff
@@ -0,0 +1,45 @@
+package life.qbic.datamanager.views.layouts;
+
+import com.vaadin.flow.component.Component;
+import com.vaadin.flow.component.Composite;
+import com.vaadin.flow.component.html.Div;
+import com.vaadin.flow.component.html.Span;
+
+/**
+ * <b>Card, A {@link Composite} component containing a {@link Div} hosting the content to be shown
+ * on a card within a {@link PageComponent} in the data-manager-app</b>
+ */
+public class Card extends Composite<Div> {
+
+  private final Span title = new Span("");
+
+  public Card() {
+    add(title);
+    setDefaultCardStyles();
+  }
+
+  public Card(String title, Component... content) {
+    this.title.setText(title);
+    getContent().add(content);
+    add(this.title);
+    setDefaultCardStyles();
+  }
```

Comment:
I thought a simple card is empty and provides only border and shadow?

Reply:
I'd have assumed that the Card needs a way to host the content we want to show within the card. This constructor is just a more convienient way to set the title and content during the card creation instead of setting the title and content later. I can remove the constructor if you wish 👍